Lasse/LittleWindows.asm

```diff
@@ -279,8 +280,7 @@ MainEntry:
         push esp                                ; Pointer to struct
         call dword ptr[ebp + fn_GetStartupInfoA]
         lea eax, (STARTUPINFOA ptr[esp]).wShowWindow
-        mov ecx, 1
-        test eax, ecx                           ; Find out if wShowWindow should be used
+        test al, 1                              ; Find out if wShowWindow should be used

```

Comment:
I'm not clear if there's a need to set `ecx` to `1` right here, so if you're only using it to check the low bit of `eax` then a `test al, 1` should be all you need.  

If you do need `ecx` to be `1` hereafter, then `xor ecx, ecx`, `inc ecx` will still save 2 bytes.